packages/admin-panel-server/src/routes/index.ts

```diff
@@ -8,7 +8,6 @@ export * from './dashboardVisualisations';
 export * from './mapOverlayVisualisations';
 export * from './FetchHierarchyEntitiesRoute';
 export * from './FetchReportPreviewDataRoute';
-export * from './FetchAggregationOptionsRoute';
```

Comment:
Removed all the 'FetchAggregationOptions' stuff as there's no longer a UI for the fetch config. This is a bit of a step backwards in terms of viz-builder functionality, which is sad, but I think things will return better than before soon enough when we convert the transform steps to having a complete UI 👍